### PR TITLE
adding vagrant installation

### PR DESCRIPTION
**Pull request by pYr0x**
please check for typing error

### CODE REVIEW

File: docs/setting-up.md [hunk 1]
```diff
@@ -231,3 +231,69 @@ Then click **OK** to create the device.
 
 Close the AVD Manager and you should have everything you need for Android development.
 
+
+## Vagrant & VritualBox
```
Comment (daffl) [inline docs/setting-up.md]: Typo ;)

**Comment by daffl** [comment]
I think we can get this one in. I'll double check again but last time everything worked as described. Is MongoDB still enabled? I think we can remove it since it is not needed for DoneJS and conflicts with existing MongoDB installations.

**Comment by pYr0x** [comment]
Yes it is still installed. 
We can remove it, but I think people are happy that a database is pre installed and they can coding apps with donejs and a database at the backend. 

As a feature request, maybe I we can let the user choose if he want to install MongoDB or not

**Comment by pYr0x** [comment]
@daffl can we merge that ?

**Pull request merged**